rapidsai/devel-centos.Dockerfile

```diff
@@ -102,6 +102,12 @@ RUN gpuci_conda_retry install -y \
       codecov \
       mamba \
       rapids-scout-local
+# Install sccache
+RUN wget https://github.com/mozilla/sccache/releases/download/v0.2.15/sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz \
+    && tar -xzf sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz \
+    && mv sccache-v0.2.15-x86_64-unknown-linux-musl/sccache /usr/local/bin \
+    && rm -rf sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz sccache-v0.2.15-x86_64-unknown-linux-musl/ \
+    && chmod +x /usr/local/bin/sccache
```

Comment:
```suggestion
# Install sccache
ARG SCCACHE_VERSION=0.2.15
RUN wget https://github.com/mozilla/sccache/releases/download/v${SCCACHE_VERSION}/sccache-v${SCCACHE_VERSION}-x86_64-unknown-linux-musl.tar.gz \
    && tar -xzf sccache-v${SCCACHE_VERSION}-x86_64-unknown-linux-musl.tar.gz \
    && mv sccache-v${SCCACHE_VERSION}-x86_64-unknown-linux-musl/sccache /usr/local/bin \
    && rm -rf sccache-v${SCCACHE_VERSION}-x86_64-unknown-linux-musl.tar.gz sccache-v${SCCACHE_VERSION}-x86_64-unknown-linux-musl/ \
    && chmod +x /usr/local/bin/sccache
```

Last suggestion. We can a Dockerfile `ARG` to make updating the `sccache` version easier in the future.